docs/apk-generator.md

```diff
@@ -1,18 +1,18 @@
-## How apk is generated using the Android apk generator
+# How apk is generated using the Android apk generator
 
 ## **Getting the data from uploaded zip/url**
 
-![Apk Generator](images/androidgenerator.png)
+![Apk Generator](_static/images/androidgenerator.png)
```

Comment:
@pri22296 I thought keeping assets in two different folders won't be looking good. so, I kept all the images in `_static` folder. even it looks consistency